pkg/buildkitutil/buildkitutil.go

```diff
@@ -143,9 +130,21 @@ func PingBKDaemon(buildkitHost string) error {
 	return nil
 }
 
+// contains open-codes slices.Contains (without generics) from Go 1.21.
```

Comment:
Nit: Typo: "open-coded". >_<

Reply:
Do you plan to fix this before merging?

Reply:
Ooops, looking at it again, the comment is actually correct: `contains` is the name of the function.